shardingsphere-mode/shardingsphere-mode-core/src/test/java/org/apache/shardingsphere/mode/metadata/MetaDataContextsFactoryTest.java

```diff
@@ -53,18 +70,57 @@ public final class MetaDataContextsFactoryTest {
     
     @Mock
     private DatabaseMetaDataPersistService databaseMetaDataPersistService;
-    
+
+    @Mock
+    private ShardingSphereDatabase mockShardingSphereDb;
```

Comment:
Do you think database is better?

Reply:
Makes sense. done, https://github.com/apache/shardingsphere/pull/20778/commits/802fc58ebc48f7e41ca1281a3fb7708442be5b32#diff-4d5951cc0f1eb8b3f9a46f67750ce90b78ca6d217d9f8e69c1e35985abf249e3R75